tests/utils.spec.js

```diff
@@ -1,6 +1,14 @@
-import { sortObjectKeys } from '../src/utils'
+import {graphQlSanitize, sortObjectKeys} from '../src/utils'
 
 describe('utils', () => {
+  describe('graphQlSanitize', () => {
+    const input = "{\n      pair\t (id: \"0x81e11a9374033d11cc7e7485a7192ae37d0795d6\") {\n        token1Price\r      }"
+    const expected = "{pair(id:\"0x81e11a9374033d11cc7e7485a7192ae37d0795d6\"){token1Price}"
+    const result = graphQlSanitize(input)
+
+    expect(result).toStrictEqual(expected)
+  })
+
```

Comment:
Could you add a test for a query with multiple fields? I think we can't remove all the spaces

```js
    const input = "{\n      pair\t (id: \"0x81e11a9374033d11cc7e7485a7192ae37d0795d6\") {\n        token1Price\n        token0Price\r      }"
    const expected = "{pair(id:\"0x81e11a9374033d11cc7e7485a7192ae37d0795d6\"){token1Price token0Price}"
    const result = graphQlSanitize(input)

    expect(result).toStrictEqual(expected)
  })
```

Reply:
Great point.

I changed it to use `graphql-query-compress` for a more thorough sanitization, and the case above is now covered by the tests.